pandas/core/arrays/numeric.py

```diff
@@ -188,3 +189,48 @@ def reconstruct(x):
             return tuple(reconstruct(x) for x in result)
         else:
             return reconstruct(result)
+
+    def _apply(self, func: Callable, **kwargs) -> NumericArray:
```

Comment:
cc @jorisvandenbossche @jbrockmendel do we want specific EA tests?

Reply:
Not for general EA, but for the numeric arrays specifically, yes. As mentioned on the other PR, I would create a new `test_function.py` in the `/tests/arrays/masked/` dir for this)

Reply:
@jorisvandenbossche For a non-native english speaker, what does "EA" stand for?

Reply:
@jorisvandenbossche I can add a file in the `/tests/arrays/masked/`folder in order to test the consistency between the numpy `round` and the pandas `round` for all the following `FLOAT_EA_DTYPES` types. Nevertheless, I now use the `any_float_allowed_nullable_dtype` fixture in the test `test_round_numpy_with_nan` of `tests/series/methods/test_round.py`.

Reply:
can you add general EA tests

Reply:
> I can add a file in the `/tests/arrays/masked/` folder in order to test the consistency between the numpy `round` and the pandas `round` for all the following `FLOAT_EA_DTYPES` types. Nevertheless, I now use the `any_float_allowed_nullable_dtype` fixture in the test `test_round_numpy_with_nan` of `tests/series/methods/test_round.py`.

I would still add a small test in `/tests/arrays/masked/` as well. Since this is a public method on the integer and floating arrays, we should test that specifically as well (and not only through Series)

Reply:
And we should also test calling numpy.round on the array, see https://github.com/pandas-dev/pandas/pull/38866#discussion_r551800866